lcftrans/Makefile.am

```diff
@@ -0,0 +1,12 @@
+bin_PROGRAMS = lmu2png
```

Comment:
This is an oversight and needs to be changed in the whole file ;-)
i.e. `sed 's/lmu2png/lcftrans/g' -i lcftrans/Makefile.am`